code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -271,8 +271,19 @@
 				visible_message("<span class='danger'>[M] disarmed [src]!</span>", \
 						"<span class='userdanger'>[M] disarmed [src]!</span>")
 			else
+				//yogs start
+				var/obj/item/bodypart/affecting = get_bodypart(ran_zone(M.zone_selected))
+				if(!affecting)
+					affecting = get_bodypart(BODY_ZONE_CHEST)
+				var/armour = run_armor_check(affecting, "melee")
+				if(prob(armour))
+					return

```

Comment:
Perhaps add a message here so it doesn't silently do nothing

Reply:
the run_armor_check tells the person being disarmed that their armour protects them, I guess I can add one for the alien

Reply:
yeah that'd be nice